app/components/businesses/scheduling_link_component.rb

```diff
@@ -0,0 +1,18 @@
+module Businesses
+  class SchedulingLinkComponent < ApplicationComponent
+    private attr_reader :user, :conversation
+
+    def initialize(user, conversation)
+      @user = user
+      @conversation = conversation
+    end
+
+    def render?
+      user.business && scheduling_link.present?
```

Comment:
This should check if the user's business is the same as the business involved in the conversation. I think we have a method on `Conversation` to check that.

Reply:
Nice, changed that.